gci/models.py

```diff
@@ -1 +1,26 @@
-# Create your models here.
+from django.db import models
+
+class Students(models.Model):
+    identifier = models.IntegerField(default=0, primary_key=True)
+    display_name = models.CharField(max_length=100, default='')
+    email_id = models.EmailField(default='')
```

Comment:
As per @jayvdb comment https://github.com/coala/community/pull/13/files#r154513055
Please undo the db migration as well.

Reply:
Email definitely needs to be removed here.

but wrt migrations, lets keep it, we might try to persist the expensive to collect data , push it to master with `[skip ci]`, and reload it on the next run.
Lets get into the practise of doing a normal django app even for this.